src/Formik.tsx

```diff
@@ -493,7 +493,7 @@ export class Formik<ExtraProps = {}, Values = object> extends React.Component<
     );
   };
 
-  setFieldError = (field: string, message: string) => {
+  setFieldError = (field: string, message: string | undefined) => {
```

Comment:
Do we need to update setFieldValue and setFieldTouched too?

Reply:
I'm not sure - i updated the `setFieldValue` typings because of a comment of @tgriesser in #727.
Let me have a look!

Reply:
Seems we only need to update `setFieldError`.